child_compassion/models/compassion_hold.py

```diff
@@ -70,6 +70,13 @@ def check_hold_validity(self):
             ('expiration_date', '<',
              fields.Datetime.now())
         ])
+
+        for expired_hold in expired_holds:

```

Comment:
Why looping twice in the same lists when you can do all operations in the same loop?